### PR TITLE
better chinese translation 

### PR DESCRIPTION
**Pull request by lin-123**
include detail explaination

### CODE REVIEW

**Comment by lin-123** [comment]
ok, i was fixed the [chinese simplify](https://github.com/lin-123/eslint-config-airbnb-ch) of README.md

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Airbnb JavaScript Style Guide() {
```
Comment (JBallin) [inline README.md]: Don't remove parens and curly bracket

File: README.md [hunk 2]
```diff
@@ -3867,5 +3867,3 @@ SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 ## Amendments
 
 We encourage you to fork this guide and change the rules to fit your team’s style guide. Below, you may list some amendments to the style guide. This allows you to periodically update your style guide without having to deal with merge conflicts.
-
-# };
```
Comment (JBallin) [inline README.md]: Don't remove newline, hash, curly bracket, semicolon

File: README.md [hunk 3]
```diff
@@ -3811,7 +3811,7 @@ Other Style Guides
   - ![br](https://raw.githubusercontent.com/gosquared/flags/master/flags/flags/shiny/24/Brazil.png) **Brazilian Portuguese**: [armoucar/javascript-style-guide](https://github.com/armoucar/javascript-style-guide)
   - ![bg](https://raw.githubusercontent.com/gosquared/flags/master/flags/flags/shiny/24/Bulgaria.png) **Bulgarian**: [borislavvv/javascript](https://github.com/borislavvv/javascript)
   - ![ca](https://raw.githubusercontent.com/fpmweb/javascript-style-guide/master/img/catala.png) **Catalan**: [fpmweb/javascript-style-guide](https://github.com/fpmweb/javascript-style-guide)
-  - ![cn](https://raw.githubusercontent.com/gosquared/flags/master/flags/flags/shiny/24/China.png) **Chinese (Simplified)**: [yuche/javascript](https://github.com/yuche/javascript)
+  - ![cn](https://raw.githubusercontent.com/gosquared/flags/master/flags/flags/shiny/24/China.png) **Chinese (Simplified)**: [lin-123/eslint-config-airbnb-ch](https://github.com/lin-123/eslint-config-airbnb-ch)
```
Comment (JBallin) [inline README.md]: I would consider renaming your fork (and link) to just be 'javascript', for consistency.

**Comment by ljharb** [comment]
@JBallin please be sure that your translation work is saved in a branch that's not `master`; i'll need to squash and force push this before merging it.

**Comment by lin-123** [comment]
so, what should i do?  rename the repo to  https://github.com/lin-123/javascript ?

**Comment by ljharb** [comment]
@lin-123 yes, it would be ideal to rename the repo to match.

**Comment by lin-123** [comment]
@ljharb  repo was renamed. Chinese already saved in `cn`, `master` will follow upstream master. please review the translation.

**Pull request merged**